Library/include/CSP/Systems/Assets/AssetSystem.h

```diff
@@ -251,11 +252,43 @@ class CSP_API AssetSystem : public SystemBase
 	CSP_ASYNC_RESULT_WITH_PROGRESS void
 		RegisterAssetToLODChain(const AssetCollection& AssetCollection, const Asset& Asset, int LODLevel, AssetResultCallback Callback);
 
+	/// @brief Creates a new material backed by an AssetCollection/Asset.
+	/// @param Name const csp::common::String& : The name of the new material.
+	/// @param SpaceId const csp::common::String& : The space id this material is associated with.
+	/// @param Callback GLTFMaterialResultCallback : Callback when asynchronous task finishes.
+	CSP_ASYNC_RESULT void CreateMaterial(const csp::common::String& Name, const csp::common::String& SpaceId, GLTFMaterialResultCallback Callback);
+
+	/// @brief Updates an existing materials properties.
```

Comment:
[Question] I tried to google it but didn't find anything - is it customary not to use apostrophes in documentation strings? (i.e. "an existing material's properties"). I have seen this several times around the code now

Reply:
I'm no good word person, but I don't believe plurals get apostrophes unless they're possessive.

Reply:
I don't think plurals get apostrophes unless they're possessive.

Reply:
correct. however, "an" is not plural :-)

Reply:
I don't think it's customary  I just can't spell :(

Reply:
> correct. however, "an" is not plural :-)
🤦 

This is also actually possesive, because it's properties that belong to a material. I'm a stupid.